static/app/components/organizations/pageFilters/parse.spec.tsx

```diff
@@ -169,32 +169,13 @@ describe('normalizeDateTimeParams', function () {
         statsPeriod: '14d',
       })
     ).toEqual({
-      utc: 'true',
       statsPeriod: '90d',
     });
   });
 
   it('does not return default statsPeriod if `allowEmptyPeriod` option is passed', function () {
     expect(normalizeDateTimeParams({}, {allowEmptyPeriod: true})).toEqual({});
   });
-
-  it('should parse utc when it is defined', function () {
```

Comment:
I think this test is still relevant, you probably need to sent a start/end time to it though